src/components/AnnualReport/AGMDate.vue

```diff
@@ -14,27 +21,27 @@
             transition="scale-transition"
             max-width="290"
           >
-            <template v-slot:activator="{ on, attrs }">
+            <template #activator="{ on, attrs }">
```

Comment:
Did you change this or did ESLint? Could you test it, please?

Reply:
Tested, working fine!
![Screenshot (78)](https://github.com/bcgov/business-filings-ui/assets/122301442/da3dcea4-f0ac-42a0-a132-d27f59750ea3)
![tested 2](https://github.com/bcgov/business-filings-ui/assets/122301442/34db1908-6396-40f2-9d90-c7bc3542750b)